packages/bindgen/spec.yml

```diff
@@ -366,9 +366,9 @@ records:
   UserIdentity:
     cppName: SyncUserIdentity
     fields:
-      id: 
+      id:
         type: std::string
-      provider_type: 
+      provider_type:
         type: std::string
```

Comment:
FYI: when you don't need a `default` you can just use the shorthand `name: Type` like `id: std::string`.

I know this was probably just an automatic whitespace cleanup, so don't feel like you need to fix this in your commit, but I wanted to mention it in case you end up declaring a record in the future.

CC @JacobOscarGunnarsson